src/main/java/org/ltr4l/trainers/PRankTrainer.java

```diff
@@ -89,6 +102,26 @@ protected double calculateLoss(List<Query> queries) {
     return thresholds;
   }
 
+  public double[] getBestWeights() {
```

Comment:
Do you still keep "best" weights?

Reply:
No, I do not keep the best weights anymore. I will change this.